SourceCode/include/tracer/Vector.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include <ostream>
+struct Vector {
+  float x;
+  float y;
+  float z;
```

Comment:
Index the into the members of the vector is very useful, so try to use an array for its components